### ISSUE TITLE
Feature Request: Storage size limit on the new Video cache

### ISSUE THREAD

**Issue by danieldaquino** [damus-io/damus#2192]
As a Damus user who probably has lots of videos on their feed, I would like the video cache to not use too much storage.

**Is your feature request related to a problem? Please describe.**
Running out of space

**When does this problem happen?**
Probably when there are too many videos on the home tab

**Describe the solution you'd like**
Video cache should have a storage limit (e.g. 2GB)

**Describe alternatives you've considered**
Manually clear cache more frequently

**Additional context**
Video cache is a brand new feature. Will reported that the video cache consumed about 20GB on his phone even though the expiry is set to 1 day.

This is related to another ticket about storage limits on the image cache (https://github.com/damus-io/damus/issues/1554), but since the video cache is a completely different cache, it might be worth tracking it as a separate ticket.

**Comment by alltheseas** [comment]
> Video cache should have a storage limit (e.g. 2GB)

> This is related to another ticket about storage limits on the image cache (https://github.com/damus-io/damus/issues/1554), but since the video cache is a completely different cache, it might be worth tracking it as a separate ticket.

Thanks for tagging. What I most like about the telegram storage approach reference in #1554 is that there is max cache size slider, which includes everything.

**Comment by danieldaquino** [comment]
> there is max cache size slider, which includes everything

Yeah I agree, UI-wise we should have a single slider for everything, to make it easy to use.

**Comment by alltheseas** [comment]
@danieldaquino suggested video is not stored in cache in 1.13
